tests/kafkatest/tests/streams/streams_broker_bounce_test.py

```diff
@@ -216,12 +208,7 @@ def collect_results(self, sleep_time_secs):
             broker_type=["leader"],
             num_threads=[1, 3],
             sleep_time_secs=[120],
-            metadata_quorum=[quorum.isolated_kraft])
-    @matrix(failure_mode=["clean_shutdown", "hard_shutdown", "clean_bounce", "hard_bounce"],
-            broker_type=["leader", "controller"],
-            num_threads=[1, 3],
-            sleep_time_secs=[120],
-            metadata_quorum=[quorum.zk])
+            metadata_quorum=[quorum.combined_kraft])
```

Comment:
As above: should we move this parameter and hard-code?

Reply:
answered above